data/json/npcs/talk_tags.json

```diff
@@ -9,8 +9,7 @@
       "Thorazine is poison.",
       "You don't need thorazine, it's limiting you.",
       "Thorazine... That's what 'they' use to keep you tame.",
-      "Rather not cloud my mind.",
-      "No thanks. I need to stay sharp."
+      "Don't. You need to stay sharp."
```

Comment:
I think it needs to mention thorazine, otherwise it's going to be pretty unclear what they're talking about.

Reply:
Fixed.